docs/tutorials/fit_convergence.rst

```diff
@@ -17,26 +17,25 @@ of AS 209 from the Quickstart, looks like this,
 
 **a)** The fitted frank brightness profile over all fit iterations.
 Note how small amplitude, fast oscillations that are due to unconstrained
-baselines are damped over the first :math:`\approx 300` iterations.
+baselines are damped over the first :math:`\approx 100` iterations.
 The fit runs until a convergence criterion on the power spectrum is met at every collocation point,
 :math:`|(P_{\rm i} - P_{\rm i-1}| \leq {\rm tol} * \pi`,
 where :math:`P_{\rm i}` is the power spectrum at iteration :math:`i`
 and :math:`{\rm tol}` is the tolerance (`iter_tol`) in your parameter file.
-This criterion is more robust than one based on the brightness profile because of the oscillations imposed on the latter by sparse sampling.
-If this stopping condition is not met, the fit runs until `max_iter` as set in your parameter file. |br|
+This criterion using the power spectrum is more robust than one based on the brightness profile because of the oscillations imposed on the latter by sparse sampling.
+If this stopping condition is *not* met, the fit runs until `max_iter` as set in your parameter file. |br|
 **b)** Sequential difference between the last 100 brightness profile iterations.
-Note the y-scale here is in :math:`10^5\ {\rm Jy\ sr}^{-1}` (compared to :math:`10^{10}\ {\rm Jy\ sr}^{-1}` in (a)),
-so in this case the oscillations remaining at the end of the fit (:math:`\approx 1250` iterations) are at parts in :math:`10^6`.
+The oscillations remaining at the end of the fit (:math:`\approx 630` iterations) are at a couple parts in :math:`10^5`.
 |br|
 **c)** The reconstructed power spectrum over all fit iterations.
-Our initial guess for the power spectrum, a power law with slope of -2, is apparent in the longest baselines for the first :math:`\approx 250` iterations,
+Our initial guess for the power spectrum, a power law with slope of -2, is apparent in the longest baselines for the first :math:`\approx 100` iterations,
```

Comment:
I think you should point out that the number of iterations is \alpha dependent

Reply:
Good point, added in #700d119